tests/api_test.py

```diff
@@ -24,25 +22,24 @@ def test_basepath_rewriting():
 def build_config(schema_dir):
     return mock.Mock(
         registry=get_registry({
-            'swagger_schema': compile_swagger_schema(schema_dir),
+            'swagger_schema': validate_swagger_schema(schema_dir),
         })
```

Comment:
I don't think this is right `validate_swagger_schema` doesn't return anything. This should still be `compile_swagger_schema`.

`pyramid_swagger` should still return the same exceptions, we probably just want to wrap the exception